pkg/util/compat/jsoncopy.go

```diff
@@ -0,0 +1,18 @@
+package compat
```

Comment:
[q] what does `compat` mean? :)

Reply:
compatibility, reason being "stuff we need to be compatible with `mongodbatlas`"